ansible-st2-local/roles/arteria/tasks/conf.yml

```diff
@@ -1,17 +0,0 @@
----
-
-# TODO: Change this user name in a better way in the future..
-- name: Change the system user from the default stanley to our provision user
-  sudo: yes
-  replace: regexp="^user = stanley$" replace="user = {{ ansible_ssh_user }}" dest=/etc/st2/st2.conf backup=yes
```

Comment:
That's probably a wise choice if we want `arteria` to be the user that does the remote commands.